README.md

```diff
@@ -20,3 +20,77 @@ The following platforms are officially supported (tested):
 
 If you want to know how to build this project and contribute to it, please
 check out the [Contributing Guide](CONTRIBUTING.md).
+
+
+## Usage
+
+Please also refer to [examples](examples) for more detailed usage.
+
+### Installation
+
+```bash
+# Choose the version to install
+VERSION=0.2.0
+pip install frequenz-client-reporting==$VERSION
+```
+
+
+### Initialize the client
```

Comment:
Nitpick: We usually leave an empty line after headings:

```suggestion
### Initialize the client

```

Same for the other headings...